setup.py

```diff
@@ -21,7 +21,7 @@
 import platform
 import io
 
-__version__ = "0.9.3"
+__version__ = "0.9.4.rc0"
```

Comment:
Once we're satisfied with testing https://github.com/Everlaw/servers/pull/39905, we should remove the `.rc0` suffix.